decidim-proposals/app/cells/decidim/proposals/participatory_text_proposal_cell.rb

```diff
@@ -30,7 +30,7 @@ def title
 
       def body
         return unless model.participatory_text_level == "article"
-        present(model).body
+        simple_format present(model).body(links: true)
```

Comment:
Would not it be better to use the `decidim_sanitize` method instead of the `simple_format`?